workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/WorkflowNode.java

```diff
@@ -60,4 +60,8 @@ public boolean isConditional(String parentId) {
   public String getIfCondition(String parentId) {
     return this.ifConditions.get(parentId);
   }
+
+  public boolean isNotExclusiveFormReply() {
+    return getElementType() == WorkflowNodeType.FORM_REPLIED_EVENT && !getEvent().getFormReplied().getExclusive();
```

Comment:
Can this sometimes cause a NP? if getEvent().getFormReplied() is null for instance..

Reply:
I would call it isExclusiveFormReply() and check it returns false

Reply:
if the type is a form reply, getEvent().getFormReplied() won't be null.
I call it isNotExclusiveFormReply, it s because everywhere this function is called, it is to check the form to be not exclusive, so it is more convenient to not have "!" every time this is called.

Reply:
Yes but if one day we want to check the form to be exclusive, we will call (!isNot..) and you read it (not is not exclusive..), it will be mind blowing ^^